bindings/java/src/main/com/apple/foundationdb/TenantManagement.java

```diff
@@ -52,6 +52,10 @@ public class TenantManagement {
 	 * @param tenantName The name of the tenant. Can be any byte string that does not begin a 0xFF byte.
 	 */
 	public static void createTenant(Transaction tr, byte[] tenantName) {
+		if (FDB.instance().getAPIVersion() < 720) {
```

Comment:
Better use here one of the newly introduced named API features. They are still not available for bindings but Dennis is working on it.

Reply:
Fair enough, if that's imminent we can wait. Otherwise I'd say we should go ahead and merge this and then include the change in Dennis's PR.